Lombiq.Npm.Targets.targets

```diff
@@ -63,13 +63,105 @@
            AlwaysCreate="true" />
   </Target>
 
+  <UsingTask TaskName="CopyVendors"
```

Comment:
@Piedone Can I get a an early impression about this approach? I know a big blob of code-as-text is not the prettiest, but I couldn't find a better approach without an explosion in implementation complexity. If you have any suggestions I'm all ears.

On the flip side, check out how simple and elegant [the usage](https://github.com/Lombiq/Orchard-Chart.js/compare/issues%2FOSOE-1107?title=issues%2FOSOE-1107&quick_pull=1#diff-f91e8cb13020d27ff099c6aa99668c85e68a9e92bcb795ff50343b20067d6ee3R27-R29) is in a consuming library. I think the end result makes it well worth it. (also this is still way more readable to me than the average MSBuild target spaghetti)

Reply:
Oh, interesting!